docs/REFERENCE.md

```diff
@@ -218,6 +218,31 @@ const fetchDataOrErrorResult = Result.from(async () => {
 
 ---
 
+## Resultify
+`Resultify` is helper function very similar in work to `Promisify`. It changes return type of function to Result.
```

Comment:
It won't be obvious for every dev what `Promisify` is. Plz add link to https://nodejs.org/api/util.html#utilpromisifyoriginal and call it `util.promisify`